### PR TITLE
Fix downloading build artifact in case build is triggered from forked…

### PR DESCRIPTION
**Pull request by omeshp**
… github repos.

### CODE REVIEW

File: Tasks/DownloadBuildArtifactsV0/task.json [hunk 1]
```diff
@@ -239,6 +239,7 @@
         "DownloadArtifacts": "Downloading artifact %s from: %s",
         "DownloadingArtifactsForBuild": "Downloading artifacts for build: %s",
         "LinkedArtifactCount": "Linked artifacts count:  %s",
+        "ExtractionFailed": "Failed to extract package with error %s",
```
Comment (Lovakumar) [inline Tasks/DownloadBuildArtifactsV0/task.json]: Package extraction failed with error %s

Comment (omeshp) [reply Tasks/DownloadBuildArtifactsV0/task.json]: will take separately

**Comment by rozele** [comment]
I'm currently getting 404s when using the DownloadBuildArtifacts task in my Azure Pipeline:
https://dev.azure.com/NLUDevOps/NLU.DevOps/_build/results?buildId=62

The build is triggered by a PR from a fork. Does this PR resolve the issue, and if so any ETA on when it will be pushed to production?

**Comment by Lovakumar** [comment]
> I'm currently getting 404s when using the DownloadBuildArtifacts task in my Azure Pipeline:
> https://dev.azure.com/NLUDevOps/NLU.DevOps/_build/results?buildId=62
> 
> The build is triggered by a PR from a fork on GitHub. Does this PR resolve the issue, and if so any ETA on when it will be pushed to production?

This fix is already in production. Please try and let us know if it's not working for you.

**Comment by rozele** [comment]
Thanks @Lovakumar - sent you mail.

**Pull request merged**